ShopLens/ShopLensForms/Controllers/MainController.cs

```diff
@@ -42,19 +53,31 @@ public void StartApp()
 
         public void StartVoiceRecognizer()
         {
-            //Register commands to voice recognizer and register grammar events to methods
-            //while the form loads.
+            _voiceRecognizer.AddCommand(helloCmd, CommandRecognized_Hello);
             _voiceRecognizer.AddCommand(whatIsThisCmd, CommandRecognized_WhatIsThis);
             _voiceRecognizer.AddCommand(startCmd, CommandRecognized_Start);
             _voiceRecognizer.AddCommand(exitCmd, CommandRecognized_Exit);
             _voiceRecognizer.StartVoiceRecognition();
         }
 
-        /// <summary> Calls method when someone says "what is this" </summary>
+        /// <summary> Calls method when someone says a certain command. </summary>
         /// <remarks>
-        /// This if statement makes sure the <see cref="WhatIsThis_btn_Click"/>
-        /// is called within the GUI thread. For information see https://stackoverflow.com/a/10170699
+        /// This if statement makes sure the method that must be executed when the specified command is recognized
```

Comment:
Should we specify what the method does if the user won't be able to see the body of the method because it's private?

Reply:
Doesn't it seem like we're giving away the ㊙️ secrets ㊙️ of our trade?

Reply:
I don't think documentation on private methods is exposed to the users.

So we're basically helping each other out and no one else will see them, so I think it's OK in this case 👌

Reply:
Ok, but still should I leave it like this, because right now I can just write inheritdoc on similar methods or rewrite it your way and then write a unique summary comment for every similar method with a different cref link?